libs/common/src/vault/services/fido2/fido2-authenticator.service.ts

```diff
@@ -136,6 +136,14 @@ export class Fido2AuthenticatorService implements Fido2AuthenticatorServiceAbstr
         cipher = await encrypted.decrypt(
           await this.cipherService.getKeyForCipherKeyDecryption(encrypted)
         );
+
+        if (cipher.reprompt !== 0 && !userVerified) {
```

Comment:
issue: avoid magic numbers
```suggestion
        if (cipher.reprompt !== CipherRepromptType.None && !userVerified) {
```